### ISSUE TITLE
[Win][Fx108] Non-tab content in tab bar gets collapsed with tabs below location bar

### ISSUE THREAD

**Issue by artemiswkearney** [Aris-t2/CustomCSSforFx#513]
1. Issue description:

With `@import "./css/tabs/tabs_below_navigation_toolbar.css";` enabled, all non-tab content in the tab bar (including new tab button and tab-list dropdown button) gets collapsed. Adding `my_userChrome.css` with: 
```css
#tabbrowser-tabs {
  width: unset;
}
```
fixes this. (The issue also doesn't occur with the `_alt` version or the `_macOS` version of the option, but those have other downsides.)
(Expected behavior: content added to the tab bar with customize mode remains visible after leaving customize mode)

2. Does the issue occur with latest CustomCSSforFx release?
(Check here: https://github.com/Aris-t2/CustomCSSforFx/releases)

Yes

3. Steps to reproduce the issue:

- Install latest version of CustomCSSforFx (4.2.5v2 at time of writing) 
- Uncomment `@import "./css/tabs/tabs_below_navigation_toolbar.css";` on line 570 of userChrome.css
- In customize mode, place content such as addon buttons on the right side of the tab bar
- After leaving customize mode, notice that the only thing in the tab bar is tabs (even if there aren't enough tabs to fill the width of the window)
- To fix: Create `my_userChrome.css` with content: 
```css
#tabbrowser-tabs {
  width: unset;
}
```

4. SCREENSHOT(S) of the issue (drag & drop images into this post):

before enabling tabs below navigation toolbar:
![customcss_default](https://user-images.githubusercontent.com/19716820/204109726-8b8a556b-2fca-4b65-9e72-0470cfde0c52.png)

after enabling:
![customcss_below_navigation_toolbar_default](https://user-images.githubusercontent.com/19716820/204109733-5af47a6c-dcce-4544-b2ae-8eae5ed35014.png)

after adding `#tabbrowser-tabs { width: unset; }`:
![customcss_below_navigation_toolbar_fixed](https://user-images.githubusercontent.com/19716820/204109755-612df580-5b71-4a14-afcc-4c42eba90ce3.png)

5. Is 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets' set to 'true'?
(Custom CSS files will only work, if this pref is set to true!)
Yes

6. System information
- OS & OS version: Windows 11 Home, 22H2, build 22621.819
- DPI / HiDPI (e.g. 100%, 125%...): 125%
- Firefox version: 108.0b6 (64-bit)
- Firefox theme (e.g. default, dark, 'theme name & link'): Light v1.2 (Mozilla) (default, I think?)
- This projects userChrome.css/userContent.css 'settings' the issue occurs with: All default except `@import "./css/tabs/tabs_below_navigation_toolbar.css";` enabled on line 570 of `userChrome.css`

**Comment by Aris-t2** [comment]
Thanks for the note.

**Comment by Aris-t2** [comment]
Fixed on [4.2.6](https://github.com/Aris-t2/CustomCSSforFx/releases).
